shell/deepdive-run

```diff
@@ -91,6 +91,15 @@ fullConfig=$run_dir/deepdive.conf
     ddlog compile "${ddlogFiles[@]}"
     export PIPELINE=  # XXX ddlog shouldn't emit this
     : ${Pipeline:=endtoend}
+
+    # set PARALLELISM env var, use max parallelism if the variable is not set
+    if [[ $(uname) = 'Linux' ]]; then
```

Comment:
Checking if `nproc` or `sysctl` is available makes more sense than relying on uname.  You could chain the options with something like:
```bash
: ${PARALLELISM:=$({
    # Linux typically has coreutils which includes nproc
    nproc ||
    # OS X
    sysctl -n hw.ncpu ||
    # fall back to 1
    echo 1
} 2>/dev/null)}
```